cmd/convox/services.go

```diff
@@ -298,6 +305,34 @@ func cmdServiceInfo(c *cli.Context) error {
 	return nil
 }
 
+func cmdServiceURL(c *cli.Context) error {
+	if len(c.Args()) != 1 {
+		stdcli.Usage(c, "url")
+		return nil
+	}
+
+	name := c.Args()[0]
+
+	service, err := rackClient(c).GetService(name)
+	if err != nil {
+		return stdcli.ExitError(err)
+	}
+
+	if service.Status == "failed" {
+		serviceFailureError := fmt.Errorf("Service failure for %s", service.StatusReason)
```

Comment:
You can probably even one-liner these with:
`return stdcli.ExitError(fmt.Errorf(...))`